archetypes/blog.md

```diff
@@ -33,7 +33,7 @@ directory. Put the images into the same directory.
 If you have an image stored at `content/en/{{ .File.Dir }}imagename.png`, you
 can reference them like the following:
 
-![Provide a good image description for improved accessibility](imagename.png)
+`![Provide a good image description for improved accessibility](imagename.png)`
```

Comment:
```suggestion
![Provide a good image description for improved accessibility](imagename.png)
```

I disagree with this change, although this will show up as a broken image, the intend is that if someone runs the hugo new command they see how an image will be used.

Reply:
Maybe have both, so that users who may be unfamiliar with images in Markdown can see what it would look like?

Reply:
> Maybe have both, so that users who may be unfamiliar with images in Markdown can see what it would look like?

Not sure if I follow. Users can see how it looks like by taking a look at the file & the preview? We can add a dummy image here to _unbreak_ it and make it clearer by that